src/models/ReproConfig.ts

```diff
@@ -22,10 +24,51 @@ export interface ReproConfig {
    */
   sessionReplay?: ReproStepsMode;
 
+  /**
+   * Repro steps mode for ANR errors on android.
+   *
+   * @default ReproStepsMode.enabled
+   */
+  anr?: ReproStepsMode;
+
+  /**
+   * Repro steps mode for AppHangs report.
+   *
+   * @default ReproStepsMode.enabled
+   */
+  appHangs?: ReproStepsMode;
+
+  /**
+   * Repro steps mode for Fatal crashes.
+   *
+   * @default ReproStepsMode.enabled
+   */
+  fatalCrash?: ReproStepsMode;
+  /**
+   * Repro steps mode for non-fatal crashes.
+   *
+   * @default ReproStepsMode.enabled
+   */
+  nonFatalCrash?: ReproStepsMode;
+
+  /**
+   * Repro steps mode for force restart crashes.
+   *
+   * @default ReproStepsMode.enabled
+   */
+  forceRestart?: ReproStepsMode;
+
+  /**
+   * Repro steps mode for outOfMemory crashes on ios.
+   *
+   * @default ReproStepsMode.enabled
+   */
```

Comment:
Are we sure all these options default to `ReproStepsMode.enabled`? I'd think they should default to `ReproStepsMode.enabledWithNoScreenshots` as crashes, can you please confirm?